PHPCI/Languages/lang.en.php

```diff
@@ -113,6 +113,7 @@
                                 (if you cannot add a phpci.yml file in the project repository)',
     'default_branch' => 'Default branch name',
     'allow_public_status' => 'Enable public status page and image for this project?',
+    'archived' => 'Archived',
```

Comment:
There is a new language string here, maybe is a good idea to add it also to other languages, so translators can see it and update it if required, @dancryer what do you think?